src/strategies/morpho-delegation/index.ts

```diff
@@ -0,0 +1,34 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'Jean-Grimal';
+export const version = '0.1.1';
+
+const abi = [
+  'function delegatedVotingPower(address account) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'delegatedVotingPower', [address])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, delegatedVotingPower]) => [
+      address,
+      parseFloat(formatUnits(delegatedVotingPower, options.decimals))
+    ])
+  );
+}
```

Comment:
Hi @Jean-Grimal You can use the `contract-call` strategy in your space, we don't need to create a new strategy, you tried it on your space?

Reply:
Hi
Mb I wasn't aware of the `contract-call` strategy. It seems to work for us.

Reply:
I tested it works